File: src/main/java/no/nav/tag/dittNavArbeidsgiver/models/AltinnTilgangssøknad.java
package no.nav.tag.dittNavArbeidsgiver.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
public class AltinnTilgangssøknad {
    String orgnr;
    String serviceCode;
    Integer serviceEdition;
    String status;
    String cratedDateTime;
    String lastChangedDateTime;
    String submitUrl;
}

File: src/main/kotlin/no/nav/arbeidsgiver/min_side/tilgangssoknad/AltinnTilgangssøknad.kt
package no.nav.arbeidsgiver.min_side.tilgangssoknad

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonInclude

@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)
data class AltinnTilgangssøknad(
    val orgnr: String? = null,
    val serviceCode: String? = null,
    val serviceEdition: Int? = null,
    val status: String? = null,
    val createdDateTime: String? = null,
    val lastChangedDateTime: String? = null,
    val submitUrl: String? = null
)